source/reference/minio-server/settings/kes.rst

```diff
@@ -0,0 +1,57 @@
+.. _minio-server-envvar-kes:
+
+===============================
+Key Encryption Service Settings
+===============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The following environment variables control behavior for the Key Encryption Service (KES) for managing encryption and keys.
```

Comment:
I think we need to revise this.

We're controlling how MinIO interacts with a KES deployment. This has nothing to do with the KES behavior itself. Especially when stateful kes drops I want to be careful to avoid users thinking configuring MinIO also configures sKES

Reply:
Updated the wording. This is a good point.